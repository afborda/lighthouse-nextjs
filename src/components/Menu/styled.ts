import styled from 'styled-components'

export const Container = styled.div`
   margin-top: 5px;
   height: 60px;
   border-bottom-left-radius: 8px;
   border-bottom-right-radius: 8px;
   background-color: #ffffff;

   .background-menu {
      display: flex;
      justify-content: space-around;
      align-items: center;
      height: 100%;
      div {
         width: 20%;
      }
   }

   .link {
      font-weight: bold;
      height: 100%;
      text-transform: uppercase;
      display: flex;
      justify-content: center;
      align-items: center;
      color: #4fc0e0;
      padding: 22px;
   }
`
