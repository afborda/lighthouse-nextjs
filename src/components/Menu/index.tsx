import React from 'react'

import { Container } from './styled'

const Menu = () => {
   return (
      <Container>
         <div className="background-menu">
            <div>
               <a className="link" href="#">
                  Início
               </a>
            </div>
            <div>
               <a className="link" href="#">
                  Combos Promocionais
               </a>
            </div>
            <div>
               <a className="link" href="#">
                  Serviços
               </a>
            </div>
         </div>
      </Container>
   )
}

export default Menu
