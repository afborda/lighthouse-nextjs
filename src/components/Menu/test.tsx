import { render, screen } from '@testing-library/react'
import Menu from '.'

describe('<Main/>', () => {
   it('should render the heading', () => {
      const { container } = render(<Menu />)

      expect(container.firstChild).toMatchSnapshot()
   })
})
