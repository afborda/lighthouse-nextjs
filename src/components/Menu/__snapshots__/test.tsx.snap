// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render the heading 1`] = `
<div
  class="styled__Container-cqlgyt-0 enQcKE"
>
  <div
    class="background-menu"
  >
    <div>
      <a
        class="link"
        href="#"
      >
        Início
      </a>
    </div>
    <div>
      <a
        class="link"
        href="#"
      >
        Combos Promocionais
      </a>
    </div>
    <div>
      <a
        class="link"
        href="#"
      >
        Serviços
      </a>
    </div>
  </div>
</div>
`;
